docs/transpile/custom-transpiler-pass.ipynb

```diff
@@ -20,7 +20,7 @@
     "    Background: DAG representation\n",
     "  </summary>\n",
     "\n",
-    "Before building a pass, it is important to introduce the internal representation of quantum circuits in Qiskit, the [directed acyclic graph (DAG)](../api/qiskit/qiskit.dagcircuit.DAGCircuit) (see [this tutorial](https://qiskit.org/ecosystem/rustworkx/tutorial/dags.html) for an overview).  To follow these steps, install the `graphivz` library for the DAG plotting functions.  Use a Python package manager (such as `pip` or `conda`) to install `pydot` and your system's native package manager (for example, `apt`, `brew`, `yum`, or `dnf`) for `graphivz`.\n",
+    "Before building a pass, it is important to introduce the internal representation of quantum circuits in Qiskit, the [directed acyclic graph (DAG)](../api/qiskit/qiskit.dagcircuit.DAGCircuit) (see [this tutorial](https://qiskit.org/ecosystem/rustworkx/tutorial/dags.html) for an overview).  To follow these steps, install the `graphviz` library for the DAG plotting functions.  Use a Python package manager (such as `pip` or `conda`) to install `pydot` and your system's native package manager (for example, `apt`, `brew`, `yum`, or `dnf`) for `graphviz`.\n",
```

Comment:
I might just point them to https://graphviz.org/download/ For windows 10, you have to download the package, add stuff to your path, use pip/conda... it's a mess.  But none of it uses apt, brew, yum, or dnf for me :)